.travis.yml

```diff
@@ -7,17 +7,17 @@ env:
   - DO_LINT=true DO_UNIT=true DO_FUNCTIONAL=false
   - DO_LINT=false DO_UNIT=false DO_FUNCTIONAL=true
```

Comment:
Are we going to double-run the unit tests (serially and via clusterrunner) for a while and let it back in?